out/production/resources/logback.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It's better to not commit this folder content. Let's add it to the gitignore file to be safe.

Reply:
Like this?
```
# out files
/out/*
```

Reply:
This should work but the common way to ignore a folder and its content is
```
# out files
out/
```

Reply:
Even though the folder was added to the gitignore, it need to be deleted manually.